client-js/app/elements/labrad-plot.ts

```diff
@@ -6,7 +6,7 @@ import {viridisData} from '../scripts/colormaps';
 /**
  * Colors for traces in 1D plots
  */
-var COLOR_LIST = ['red', 'blue', 'yellow', 'green', 'magenta'];
+var COLOR_LIST = ['blue', 'red', '#00CC00', '#DDDD00', '#DD00DD', '#0088DD'];
```

Comment:
Fixed whitespace around operands. Agreed on colorschemes. For now I modified this to match what the grapher does, but of course we can improve on that.